realm/realm-library/src/main/java/io/realm/internal/Table.java

```diff
@@ -629,6 +655,25 @@ void checkIntValueIsLegal(long columnIndex, long rowToUpdate, long value) {
         }
     }
 
+    // check if it is ok to use null value for given row and column.
+    void checkNullValueIsLegal(long columnIndex, long rowToUpdate) {
```

Comment:
The main purpose of this is to check if there is a duplicated `null` as a PK value for `UncheckedRow.setString()`/`setNull()` & `Table.setString()`. It's now renamed as `checkDuplicatedNullForPrimaryKeyValue()`